packages/content-type-read-receipt/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@xmtp/content-type-read-receipt",
+  "version": "1.0.0",
+  "description": "An XMTP content type to support read receipts",
+  "author": "XMTP Labs <eng@xmtp.com>",
+  "license": "MIT",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "browser": "dist/web/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "browser": "./dist/web/index.js",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    }
+  },
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:xmtp/xmtp-js-content-types.git",
+    "directory": "packages/content-type-read-receipt"
+  },
+  "homepage": "https://github.com/xmtp/xmtp-js-content-types",
+  "bugs": {
+    "url": "https://github.com/xmtp/xmtp-js-content-types/issues"
+  },
+  "keywords": [
+    "xmtp",
+    "messaging",
+    "web3",
+    "js",
+    "ts",
+    "javascript",
+    "typescript",
+    "content-types"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 3 chrome versions",
+      "last 3 firefox versions",
+      "last 3 safari versions"
+    ]
+  },
+  "scripts": {
+    "build:node": "tsup",
+    "build:web": "tsup --platform browser --target esnext",
+    "build": "yarn clean:build && yarn build:node && yarn build:web",
+    "clean:build": "rimraf dist",
+    "clean": "rimraf .turbo node_modules && yarn clean:build",
+    "dev": "tsup --watch",
+    "format:base": "prettier --ignore-path ../../.gitignore",
+    "format:check": "yarn format:base -c .",
+    "format": "yarn format:base -w .",
+    "generate:types": "tsup --dts-only",
+    "lint": "eslint . --ignore-path ../../.gitignore",
+    "test:jsdom": "NODE_TLS_REJECT_UNAUTHORIZED=0 vitest run --environment jsdom",
+    "test:node": "NODE_TLS_REJECT_UNAUTHORIZED=0 vitest run --environment node",
+    "test": "yarn test:node && yarn test:jsdom",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@xmtp/proto": "^3.26.0",
+    "@xmtp/xmtp-js": "^9.2.0"
+  },
+  "devDependencies": {
+    "@types/node": "^18.14.2",
+    "@xmtp/content-type-remote-attachment": "workspace:*",
```

Comment:
this is not necessary here

Reply:
i just copied the replies boilerplate for this. will remove it there too

Reply:
gotcha, updated in read receipts only :)